src/cmd-coreos-prune

```diff
@@ -190,8 +190,10 @@ def main():
                                 else:
                                     print(f"No container tags to prune for build {build_id}.")
                     actions_completed.append(action)  # Append action to completed list
+        # Only add policy-cleanup for the build in builds.json if either of the action is completed
```

Comment:
since we have to push up a new upload for the linter issue let's maybe do:

```suggestion
        # Only add policy-cleanup for the build in builds.json if any
        # of the cleanup actions were completed.
```